lib/lexer.js

```diff
@@ -142,15 +144,20 @@ Lexer.prototype._output = function() {
 Lexer.prototype._embed = function() {
   var tag = this._next() + this._next()
     , ch = this._peek();
+
   if(tag == '<' + this._embedChar) {
     this._inEmbed = true;
     if(ch == '=' || ch == '-') return this._token(Token.EMBED, this._next());
     else return this.nextToken();
   } else if(tag == this._embedChar + '>') {
     this._inEmbed = false;
     return this.nextToken();
+  } else if(tag + ch == "-" + this._embedChar + '>') {
+    this._next();
```

Comment:
@kainosnoema It's working fine–it doesn't have to strip anything.

I agree we should improve the tests.